### ISSUE TITLE
[Feature Request] Highlight current buffer

### ISSUE THREAD

**Issue by jimeh** [sabof/project-explorer#22]
I'd like to have the file of the buffer that is currently active highlighted with a different color. And also when switching to another file, it would be nice if Project Explorer could at that point update the highlighted file, and even expand any directories if needed to highlight it.

**Comment by katspaugh** [comment]
@jimeh JFYI, to update the currently opened file, you can add this to your `.emacs.d`:

```Lisp
(defvar pe/update-timer nil)

(defun pe/update ()
  (interactive)
  (when pe/update-timer
    (cancel-timer pe/update-timer))
  (setq pe/update-timer
        (run-with-timer
         0.1 nil
         (lambda ()
           (project-explorer-open)
           (other-window 1)))))

(add-hook 'post-command-hook #'pe/update))
```

**Comment by sabof** [comment]
Feature implemented in #36
